cypress/integration/models/developer/applicationinventory/assessment.ts

```diff
@@ -283,10 +284,14 @@ export class Assessment extends Application {
         this.verifyFieldValue(reviewColumnSelector, "Completed");
     }
 
-    is_notStarted(): void {
+    assessment_is_notStarted(): void {
         this.verifyFieldValue(assessmentColumnSelector, "Not started");
     }
 
+    review_is_notStarted(): void {
```

Comment:
I guess there are more status that you might be checking on assessment and review like "NotStarted" "InProgress" "Completed" .

So can we create a generic method takes status(In Progress) and selector (Assessment or Review locator).
Example : 



getStatus(selector, State){

this.verifyFieldValue(selector, status);
}

Reply:
@sshveta , The plan was to do that in a follow up PR since several files that use this method had to be updated. But, I went ahead and made the changes anyway through this PR . All the tests passed locally .